package.json

```diff
@@ -39,31 +39,31 @@
 		"commands": [
 			{
 				"command": "fakesharper.inspectcode",
-				"title": "[fakesharper] Inspect Code"
+				"title": "FakeSharper: Inspect Code"
```

Comment:
```suggestion
				"title": "Inspect Code",
				"category": "FakeSharper"
```

Reply:
Ok. I will read the documentation about this topic!